src/olympia/scanners/migrations/0050_auto_20221018_1106.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.16 on 2022-10-18 11:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('scanners', '0049_auto_20221018_1043'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='scannerqueryrule',
+            name='action',
+        ),
+        migrations.RemoveField(
+            model_name='scannerqueryrule',
+            name='is_active',
+        ),
+    ]
```

Comment:
Not that it matters much, but do we want to combine these migrations into a single file?  0049 and 0050 look pretty innocuous at least.

Reply:
Yeah I kinda changed my mind on this as I was writing the patch. Started with bundling a bunch of things into migration 48, then did it piece by piece. The (admittedly weak) logic behind keeping 49 and 50 separate is that they are for different models.

Reply:
https://github.com/mozilla/addons-server/pull/19800/commits/d7fdcf4c2baebbb3576eb26a857b7c05a17c4acb merged 49 and 50